MAGFit/index.html

```diff
@@ -77,13 +103,21 @@ <h1 style="text-align:center"><a href="" style="color: #000000; text-decoration:
 </table>
 
 <table><tr><td style="width:1200px">
-    <h2 style="text-align:center">Flight Data</h2>
+    <h2 style="text-align:center">Flight Data
+        <img id="TT" src="../images/question-circle.svg" style="width: 1em; vertical-align: bottom" 
+        data-tippy-content='Zoom into a section of the flight to change the Analysis time then click "Calculate". Selecting the flying section of the log should give the results.',
+        data-tippy-maxWidth='750px'/>
+    </h2>
 </td></tr></table>
 
 <div id="FlightData" style="width:1200px;height:450px"></div>
 
 <table><tr><td style="width:1200px">
-    <h2 style="text-align:center">Expected vs measured body frame magnetic field</h2>
+    <h2 style="text-align:center">Expected vs measured body frame magnetic field
+        <img id="TT" src="../images/question-circle.svg" style="width: 1em; vertical-align: bottom" 
+        data-tippy-content='Per compass status and options, use checkboxes to select calibrations for comparison. Plots show results of calibrations in X Y and Z axis.'
```

Comment:
```suggestion
        data-tippy-content='Per compass status and options, use checkboxes to select calibrations for comparison. Plots show results of calibrations in X Y and Z axis. Normally selecting Battery 1 current/offsets and iron is optimal. De sure battery monitor is calibrated and functioning. otherwise use No Motor Comp, offsets and iron'
```

Reply:
I don't think we can make that recommendation. The default fit is just offsets and no motor comp. Really you only need more than that if your having issues.

Reply:
I have added "Ensure battery monitor is calibrated and functioning correctly" to the tip for the battery comp.

![image](https://github.com/ArduPilot/WebTools/assets/33176108/b879ae0d-fce2-49ff-b142-8d413f97f6a0)